### DISCUSSION TITLE
Could we get Merge Confidence enabled for go dependencies?

### DISCUSSION THREAD

**Discussion by vanstinator** [renovatebot/renovate#26201] [Suggest an Idea]
### Tell us more.

We use renovate across several NodeJS repos and have found the Merge Confidence scores on the renovate Pull Requests quite valuable. We're starting to ship some tooling written in go, and have missed that feature of renovate when handling dependency updates in those repos.

Could we get support for Merge Confidence tracking on go dependencies too?

Apologies if this has been posted before, I tried several search terms in both the issues and discussions in this repo and none jumped at as similar.

**Comment by rarkins** [comment]
Thanks for the feedback. We generally enable an ecosystem once we think that we have enough data for it for the results to be meaningful.

My colleague @nabeelsaabna runs this, we will chat internally and let you know

**Reply by nabeelsaabna** [reply]
@vanstinator https://github.com/renovatebot/renovate/pull/26369

**Reply by vanstinator** [reply]
Wow! Thank you so much!
